packages/seacas/libraries/ioss/src/Ioss_EntityBlock.h

```diff
@@ -88,6 +88,12 @@ namespace Ioss {
      */
     size_t get_offset() const { return idOffset; }
 
+    // Quiet comparison
```

Comment:
I think I need an explanation of why we have `equal` and `operator==` and why there is a quiet one and one that outputs the differences.

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.